lib/utils/isStandardSyntaxDeclaration.js

```diff
@@ -46,6 +46,17 @@ module.exports = function (decl) {
 		return false;
 	}
 
+	// Less map (e.g. #my-map() { myprop: red; })
+	if (
+		parent &&
+		isRule(parent) &&
+		parent.selector &&
+		parent.selector[0] === '#' &&
+		parent.selector[parent.selector.length - 1] === ')'
```

Comment:
[suggestion] Does it make sense to make the check a bit readable as below?

```suggestion
		parent.selector.startsWith('#') &&
		parent.selector.endsWith('()')
```